chromeless/chromeless.py

```diff
@@ -11,7 +11,9 @@
 class Chromeless():
     REQUIRED_SERVER_VERSION = 2
 
-    def __init__(self, gateway_url=None, gateway_apikey=None, chrome_options=None, function_name='chromeless-server-prod'):
+    def __init__(self, gateway_url=None, gateway_apikey=None, chrome_options=None,
+                 function_name='chromeless-server-prod', boto_session=boto3):
```

Comment:
Please only use immutable values such as `None` as default arguments. (I'm not sure about `boto3`'s immutability, but I'd prefer default arguments are obviously immutalbe)

Reply:
I am setting it to `boto3_session` only `session` is not that clear in my views.